dspace-server-webapp/src/test/java/org/dspace/app/rest/QATopicRestRepositoryIT.java

```diff
@@ -38,114 +41,60 @@ public class QATopicRestRepositoryIT extends AbstractControllerIntegrationTest {
 
     @Test
     public void findAllTest() throws Exception {
-        context.turnOffAuthorisationSystem();
-        parentCommunity = CommunityBuilder.createCommunity(context)
-                .withName("Parent Community")
-                .build();
-        Collection col1 = CollectionBuilder.createCollection(context, parentCommunity).withName("Collection 1").build();
-        QAEventBuilder.createTarget(context, col1, "Science and Freedom")
-                .withTopic("ENRICH/MISSING/PID")
-                .withMessage("{\"pids[0].type\":\"doi\",\"pids[0].value\":\"10.2307/2144300\"}").build();
-        QAEventBuilder.createTarget(context, col1, "Science and Freedom 2")
-                .withTopic("ENRICH/MISSING/PID")
-                .withMessage("{\"pids[0].type\":\"doi\",\"pids[0].value\":\"10.2307/2144301\"}").build();
-        QAEventBuilder.createTarget(context, col1, "Science and Freedom 3")
-                .withTopic("ENRICH/MORE/PID")
-                .withMessage("{\"pids[0].type\":\"pmid\",\"pids[0].value\":\"10.2307/2144302\"}").build();
-        QAEventBuilder.createTarget(context, col1, "Science and Freedom 4")
-                .withTopic("ENRICH/MISSING/ABSTRACT")
-                .withMessage(
-                        "{\"abstracts[0]\": \"Descrizione delle caratteristiche...\"}")
-                .build();
-        context.restoreAuthSystemState();
-        String authToken = getAuthToken(admin.getEmail(), password);
-        getClient(authToken).perform(get("/api/integration/qualityassurancetopics")).andExpect(status().isOk())
-                .andExpect(content().contentType(contentType))
-                .andExpect(jsonPath("$._embedded.qualityassurancetopics",
-                        Matchers.containsInAnyOrder(QATopicMatcher.matchQATopicEntry("ENRICH/MISSING/PID", 2),
-                                QATopicMatcher.matchQATopicEntry("ENRICH/MISSING/ABSTRACT", 1),
-                                QATopicMatcher.matchQATopicEntry("ENRICH/MORE/PID", 1))))
-                .andExpect(jsonPath("$.page.size", is(20))).andExpect(jsonPath("$.page.totalElements", is(3)));
-
-    }
-
-    @Test
-    public void findAllUnauthorizedTest() throws Exception {
```

Comment:
I apologize for having missed this comment.
These tests have been deleted because the findAll() method of QATopicRestRepository has been removed, and thus, it is no longer implemented.
**see**: https://github.com/DSpace/DSpace/pull/9267/files/d44f01d01edb5be6b9c3cccc60829cf0b6e85c09#diff-6302961cb8587940301ec3d4e25f54187cd3672540c2887900656bb821e6b28bR42
**new test**: https://github.com/DSpace/DSpace/pull/9267/files/d44f01d01edb5be6b9c3cccc60829cf0b6e85c09#diff-114b3ace354fcceecb58a85edc1925c1c8f6f544bb88a3a093d9d5a36e8af238R44